### PR TITLE
🚨 Fix TS warnings

### LINKED ISSUE #820: Validate eslint-typescript warnings
Run eslint check globally on core-react. 

* Fix minor issues which are easily fixed
* Make issues on warnings which need bigger rewrites

### PR DESCRIPTION
**Pull request by pomfrida**
Resolves #820

### CODE REVIEW

File: libraries/core-react/src/SelectionControls/Checkbox/Checkbox.tokens.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-// @ts-nocheck
```
Comment (wenche) [inline libraries/core-react/src/SelectionControls/Checkbox/Checkbox.tokens.ts:1]: 😂

**Review by wenche — APPROVED** [review]
LGTM 👍

**Comment by vnys** [comment]
Um… why is this merged? I still get 253 warnings and 3 errors in core-react 🤔

**Comment by pomfrida** [comment]
> Um… why is this merged? I still get 253 warnings and 3 errors in core-react 🤔

🤯  No tests failed. Confused.

**Comment by wenche** [comment]
> Um… why is this merged? I still get 253 warnings and 3 errors in core-react 🤔

What are you running to get these numbers? If I run  `pnpm  lint:core-react` everything is OK

**Comment by vnys** [comment]
Hm… maybe I’m screwing something up somewhere, I’ll check…

**Pull request merged**